### PR TITLE
Fix notification

### LINKED ISSUE #11: Add option to set Service title, text from React Native

### LINKED ISSUE #15: Hide Android Service when no buttons are paired
I would like to not show the notification or only if the button is connected, is it possible?

### PR DESCRIPTION
**Pull request by LucasMonteiro1**
Added option to set custom title and text to notification. (Fix #11)

---

According to https://github.com/50ButtonsEach/flic2lib-android/issues/15 show Android Service only when buttons are connected. (Fix #15)

### CODE REVIEW

**Comment by jdegger** [comment]
Hi @LucasMonteiro1 thank you for your contribution! Highly appreciated.

We will review and test the PR in a few days :) We will get back to you as soon as possible.

Thanks again

File: android/src/main/java/nl/xguard/flic2/service/Flic2Service.java [hunk 1]
```diff
@@ -55,11 +74,15 @@ public void onCreate() {
             notificationManager.createNotificationChannel(mChannel);
         }
 
-        Notification notification = new Builder(getApplicationContext(), NOTIFICATION_CHANNEL_ID)
+        notification = new Builder(getApplicationContext(), NOTIFICATION_CHANNEL_ID)
```
Comment (wesdewitte) [inline android/src/main/java/nl/xguard/flic2/service/Flic2Service.java:77]: When you don't add the meta data it sets no text at all, it should have the basic text like before. When you dont set icon is takes the icon from your app it should also do this when you dont set it specifically.

**Comment by jdegger** [comment]
Hi @LucasMonteiro1 we noticed one problem with your implementation. @wesdewitte posted a comment addressing it. Could you look into this?

Many thanks!

**Comment by jdegger** [comment]
Hi @LucasMonteiro1 your PR has been closed as we used your work in https://github.com/X-Guard/react-native-flic2/pull/19 and will merge that.

Thank you for your contribution!

**Pull request closed without merging**